matchzoo/tasks/classification.py

```diff
@@ -0,0 +1,34 @@
+"""Classification task."""
+
+import keras
+
+from matchzoo import engine
+
+
+class Classification(engine.BaseTask):
+    """Classification task."""
+
+    def __init__(self, num_classes: int = 2):
+        super().__init__()
+        if not isinstance(num_classes, int):
+            raise TypeError
+        if num_classes < 2:
+            raise ValueError
```

Comment:
Explicit is better than implicit.

Reply:
Yes, sure, please be explicit about your point.

Reply:
Raise ValueError(explicit)

Reply:
No problem.

Reply:
Just a side note, python 3+ prints the full stack trace, so it's ok to have raw exception, since these exceptions always print with the if statements here. But anyways I would like to add those too.